packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -44,6 +44,8 @@ yarn workspace v1.22.19image` is a collocated image path, this entry will be the
   };
 
   export type Author = {
+    key?: string; // TODO temporary, need refactor
```

Comment:
upcoming conflict: will be refactored when updating in https://github.com/facebook/docusaurus/pull/10216